README.md

```diff
@@ -1,63 +1,318 @@
+  </ul>
+</details>
+
+<details>
+  <summary>Data Store</summary>
+  <ul>
+    <li><a href="https://typeorm.io/">TypeORM</a></li>
+    <li><a href="https://github.com/luin/ioredis">ioredis</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Request Library</summary>
+  <ul>
+    <li><a href="https://axios-http.com/">Axios</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Scheduling</summary>
+  <ul>
+    <li><a href="https://docs.nestjs.com/techniques/task-scheduling">NestJS Schedule</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Validation</summary>
+  <ul>
+    <li><a href="https://github.com/typestack/class-validator">class-validator</a></li>
+    <li><a href="https://joi.dev/">Joi</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Environment Configuration</summary>
+  <ul>
+    <li><a href="https://github.com/motdotla/dotenv">dotenv</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Monorepo Tools</summary>
+  <ul>
+    <li><a href="https://nx.dev/">Nx</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Code Quality</summary>
+  <ul>
+    <li><a href="https://github.com/pahen/madge">Madge</a></li>
+    <li><a href="https://github.com/davidmarkclements/trace-unhandled">trace-unhandled</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Containerization</summary>
+  <ul>
+    <li><a href="https://www.docker.com/">Docker</a></li>
+    <li><a href="https://docs.docker.com/compose/">Docker Compose</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>API Documentation</summary>
+  <ul>
+    <li><a href="https://swagger.io/">Swagger</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features
+- **Account creation using [SIWF](https://github.com/AmplicaLabs/siwf)**
+  - Includes behind the scenes delegation to the provider
+- **Get User and Provider Account(s)**
+- **Handle (aka username) claiming**
+  - Create
+  - Change
+- **Keys Handling**
+  - Add keys to an account
+  - Get keys
+- **Get delegation info for an msaId**
+- **Easy integration with Web3 wallets**
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- LIVE Docs -->
+
+## 🚀 Live Docs
+
+- [Live Docs](https://amplicalabs.github.io/{gateway-service-repo})
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Nodejs](https://nodejs.org)
+- [Docker](https://docs.docker.com/get-docker/)
+
+### Environment Variables
 Modify any environment variables in the `.env` file as needed. For docker compose env `.env.docker.dev` file is used. The complete set of environment variables is documented [here](./ENVIRONMENT.md), and a sample environment file is provided [here](./env.template)
 
-### 3. Start the service:
-Run the following command to start the service:
-```bash
-docker-compose up
+### ⚡ Quick Start
```

Comment:
Quickstart has a lot of overlap with other sections. Do we want to simplify?

Reply:
We could make Quick Start be the version that just runs the account-service in a container, similar to how the SAT does it. The other steps are more for running it in a more manual development mode. We might need docker file changes to support it. 🤔